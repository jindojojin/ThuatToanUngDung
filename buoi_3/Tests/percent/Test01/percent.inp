10
9 3 20 1 67
1 1 1 1 96
19 19 28 33 1
10 0 10 70 10
10 20 20 30 20
20 20 20 20 20
25 25 0 5 45
13 7 28 12 40
90 2 2 2 4
30 20 20 30 0
