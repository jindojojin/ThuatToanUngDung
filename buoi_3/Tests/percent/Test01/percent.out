100
100
100
10
10
5
20
100
50
10
